src/Native/Runtime/unix/AsmOffsets.cpp

```diff
@@ -3,9 +3,9 @@
 // See the LICENSE file in the project root for more information.
 #ifdef _ARM_
```

Comment:
Actually, could you please just delete this ifdef? The ARM and !ARM code looks exactly the same after your change.